go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/aws/aws-sdk-go v1.44.28
 	github.com/aws/aws-sdk-go-v2/service/sts v1.16.7
 	github.com/ceph/ceph-csi/api v0.0.0-00010101000000-000000000000
-	// TODO: API for managing NFS-exports requires `ceph_ci_untested` build-tag
-	github.com/ceph/go-ceph v0.15.0
+	// TODO: API for managing NFS-exports, subvolume and snapshot metadata requires `ceph_ci_untested` build-tag
```

Comment:
ceph_ci_untested is removed for NFS API isn't it?

Reply:
I didn't know that before, thanks will fix it :+1: